package.json

```diff
@@ -42,8 +42,7 @@
     "typescript": "latest"
   },
   "dependencies": {
-    "@types/prettier": "^2.7.2",
```

Comment:
When updating the types, it said that the "types were a stub" and to use the ones shipped directly by prettier